Simplenote/Information/NoteInformationViewController.swift

```diff
@@ -154,6 +155,15 @@ private extension NoteInformationViewController {
     func configureAccessibility() {
         dismissButton.accessibilityLabel = Localization.dismissAccessibilityLabel
     }
+    
+    func configureScreenTitleLabel() {
+        screenTitleLabel.text = Localization.document
+        
+        if UIDevice.current.userInterfaceIdiom == .phone {
```

Comment:
Nitpick: we have an extension for UIDevice to check if device is iPad `UIDevice.isPad`, maybe we can use it here.

Reply:
It looks like with these changes we don't use screenTitleLabel and dismissButton at all, so I think we can safely remove the whole `headerStackView`. It wasn't used on iPad and now we're not using it on iPhone as well.

Reply:
Makes sense, updated

Reply:
Sorry I wasn't very clear. I meant that we can completely delete `headerStackView` from the code and interface file because I believe it is not used anymore.